packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -666,9 +666,7 @@ export class EditorPage {
 		//
 		// Once https://github.com/Automattic/wp-calypso/issues/54421 is resolved,
 		// this listener can be removed.
-		this.page.once( 'dialog', async ( dialog ) => {
-			await dialog.accept();
-		} );
+		this.allowLeavingWithoutSaving();
```

Comment:
Gotta love your habit of improving readability through self-documenting functions like this 👏🏻